classes/config/ConfigParser.inc.php

```diff
@@ -169,7 +169,8 @@ function updateConfig($file, $params) {
 
 				if (preg_match('/[^\w\-\/]/', $value)) {
 					// Escape strings containing non-alphanumeric characters
-					$valueString = '"' . $value . '"';
+					// Make sure no single backslash is present inside the quotes.
+					$valueString = '"' . str_replace('\\', '\\\\', $value) . '"';
```

Comment:
I think it would be better to wrap `addslashes` around `$value` -- I suspect there are other characters that need escaping, and that would take care of it symmetrically.

Reply:
Yeah, nice solution here.

Reply:
Committed: https://github.com/pkp/pkp-lib/commit/a6f72cd63d63e9031164497a17429fbd79c24fe2